bcgov/moh-prime
# PRIME-1964 Send subsequent real enrolment to manual if paper enrolment is flagged,

[PRIME-1964](https://bcgovmoh.atlassian.net/jira/software/c/projects/PRIME/boards/1?modal=detail&selectedIssue=PRIME-1964)

Please review but do not merge until the following PRs are merged: [PRIME-1954](https://bcgovmoh.atlassian.net/jira/software/c/projects/PRIME/boards/1?modal=detail&selectedIssue=PRIME-1954) & [PRIME-1900](https://bcgovmoh.atlassian.net/browse/PRIME-1900)

### How to test:

1. Create a paper enrollee with a birthdate that matches a BCSC enrollee
2. Submit and take note of the GPID
3. Flag paper enrollee as Always Manual in the adjudication actions
4. Enroll as a BCSC enrollee and provide the GPID from the step 2
5. Complete and submit enrolment
6. Login as admin and notice that the new BCSC enrollee is already flagged for Always Manual
